ui/src/components/Form/Boolean.vue

```diff
@@ -20,8 +22,25 @@ export default {
   methods: {
     getInput() {
       return this["boolean"];
+    },
+    hideEndDate()
+    {
+      if(this.formName.toLowerCase()=="workhistory")
```

Comment:
We shouldn't be hard coding things like form name into a component like this. We want to be able to reuse the behavior, not have it tied to a specific form.

Reply:
To avoid hard coding the UI validation tasks. I have defined in the People/Edit.vue a new data properties that contains the name of the form and the fields to check for validation, the type of the behavior to run (basically 2 can be only done at the DOM level element: disabled, visibility). Then the behavior is passed to the DynamicForm when it could be run for every field type. For that we need to pass the formName and the fieldName to the native component and add an handler to manage the emitted event. I have implement the case for the work history form but I commented it out the case for disabling I have added for testing purpose.